### PR TITLE
Maqs 7.0 Alpha 

### PR DESCRIPTION
**Pull request by TroyWalshProf**

### CODE REVIEW

File: Framework/BaseAppiumTest/AppiumSoftAssert.cs [hunk 1]
```diff
@@ -85,7 +49,7 @@ public override bool Assert(Action assertFunction, string assertName, string fai
 
                 if (AppiumConfig.GetSavePagesourceOnFail())
                 {
-                    AppiumUtilities.SavePageSource(this.appiumTestObject.AppiumDriver, this.appiumTestObject, StringProcessor.SafeFormatter($" ({ this.NumberOfAsserts})"));
+                    AppiumUtilities.SavePageSource(this.appiumTestObject.AppiumDriver, this.appiumTestObject, $" ({ this.NumberOfAsserts})");
```
Comment (FermJacob) [inline Framework/BaseAppiumTest/AppiumSoftAssert.cs:52]: Extra space at beginning

File: Framework/BaseDatabaseTest/ConnectionFactory.cs [hunk 2]
```diff
@@ -144,7 +143,7 @@ private static IProvider<IDbConnection> GetCustomProviderType(string providerTyp
         {
             if (string.IsNullOrWhiteSpace(providerType))
             {
-                throw new ArgumentException(StringProcessor.SafeFormatter($"Provider type is Empty"));
+                throw new ArgumentException($"Provider type is Empty");
```
Comment (FermJacob) [inline Framework/BaseDatabaseTest/ConnectionFactory.cs:146]: Don't need $

File: Framework/BaseSeleniumTest/Extensions/Find.cs [hunk 3]
```diff
@@ -131,7 +130,7 @@ public int IndexOfElementWithText(ICollection<IWebElement> list, string text, bo
             // if list size was null or empty and assert was true
             if (!list.Any() && assert)
             {
-                throw new NotFoundException(StringProcessor.SafeFormatter($"Empty or null Element Collection passed in {list?.ToString() ?? "NULL list"}"));
+                throw new NotFoundException($"Empty or null Element Collection passed in {list?.ToString() ?? "NULL list"}");
```
Comment (FermJacob) [inline Framework/BaseSeleniumTest/Extensions/Find.cs:133]: Are you performing a double null check here?

File: Framework/UtilitiesUnitTests/App.config [hunk 4]
```diff
@@ -0,0 +1,223 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```
Comment (FermJacob) [inline Framework/UtilitiesUnitTests/App.config:2]: Should we turn this into appsettings.json?

**Review by FermJacob — COMMENTED** [review]
Looking good

File: Framework/BaseAppiumTest/AppiumSoftAssert.cs (hunk 1 above)
Comment (TroyWalshProf) [reply Framework/BaseAppiumTest/AppiumSoftAssert.cs:52]: Actually want that for readability

File: Framework/BaseSeleniumTest/Extensions/Find.cs (hunk 3 above)
Comment (TroyWalshProf) [reply Framework/BaseSeleniumTest/Extensions/Find.cs:133]: Kind of but they are both required.

File: Framework/UtilitiesUnitTests/App.config (hunk 4 above)
Comment (TroyWalshProf) [reply Framework/UtilitiesUnitTests/App.config:2]: Not sure why we would change - other tests use JSON

**Review by FermJacob — APPROVED** [review]

**Pull request merged**